README.md

```diff
@@ -23,8 +23,16 @@ $ git flow release publish 0.1.0
 $ git flow release finish 0.1.0
 ```
 
-To kick off the deployment, you'll still need to push the local tags remotely
-`git push --tags`
+After you've completed the `git flow` steps, you'll need to push the changes to your local `master` and `develop` branches back to the main repository. Assuming the name of your default remote for the origin repo is `origin`, you can do that using the following commands:
```

Comment:
I would omit the last sentence because it's reasonable to have a default remote called origin that points to a different upstream. It can be assumed that any references to `origin` in this document refer to the `WikiWatershed/gwlf-e` fork.